### PR TITLE
feat: cl_ord_id fixes

### PR DESCRIPTION
**Pull request by benjaminjellis**
Sorry, I've been meaning to open this but wanted to do some more teasing. I think these fixes make using the crate more stable.

### CODE REVIEW

**Comment by Brendan-Blanchard** [comment]
This is very timely! I've been working on a branch this week to catch up with their changelog notes (skipping any v1 websockets stuff). Let me get this merged and then I'll hold off for another week or so with a release so I can get everything in the changelog implemented for one breaking change release.

File: src/wss/v2/trading_messages.rs [hunk 1]
```diff
@@ -227,7 +227,7 @@ pub struct BatchOrderParams {
 pub struct BatchCancelParams {
     pub orders: Vec<IntOrString>,
     pub token: Token,
-    #[serde(rename = "cl_ord_id")]
+    #[serde(rename = "cl_ord_id", skip_serializing_if = "Option::is_none")]
```
Comment (Brendan-Blanchard) [inline src/wss/v2/trading_messages.rs:230]: Did you run into the issue where `#[skip_serializing_none]` has to be above the `#[derive(Serialize)]` to work? It's a weird dependency and compiles (but doesn't skip it).

Comment (benjaminjellis) [reply src/wss/v2/trading_messages.rs]: oh I haven't this seems to work. is this an issue that the serde team are aware of? I searched really briefly but didn't find anything

Comment (Brendan-Blanchard) [reply src/wss/v2/trading_messages.rs]: Ah, took me a second, but I found it in their closed issues: https://github.com/jonasbb/serde_with/issues/440

Main is now up-to-date with Kraken's changelog, including these changes. I'll be testing locally this week and then I'll release a new major version assuming all goes well.

**Pull request merged**